lib/styles/modifiers/_typography.scss

```diff
@@ -46,66 +47,51 @@
 	@include textS-DEPRECATED;
 }
 
-.-textM {
+.-textM-DEPRECATED {
 	@include textM-DEPRECATED;
 }
 
-.-textL {
+.-textL-DEPRECATED {
 	@include textL-DEPRECATED;
 }
 
-.-textXL {
+.-textXL-DEPRECATED {
 	@include textXL-DEPRECATED;
 }
 
-.-headlineXXS {
+.-headlineXXS-DEPRECATED {
 	@include headlineXXS-DEPRECATED;
 }
 
-.-headlineXS {
+.-headlineXS-DEPRECATED {
 	@include headlineXS-DEPRECATED;
 }
 
-.-headlineS {
+.-headlineS-DEPRECATED {
 	@include headlineS-DEPRECATED;
 }
 
-.-headlineM {
+.-headlineM-DEPRECATED {
 	@include headlineM-DEPRECATED;
 }
 
-.-headlineL {
+.-headlineL-DEPRECATED {
 	@include headlineL-DEPRECATED;
 }
 
-.-headlineXL {
+.-headlineXL-DEPRECATED {
 	@include headlineXL-DEPRECATED;
 }
 
-.-titleS {
+.-titleS-DEPRECATED {
 	@include titleS-DEPRECATED;
 }
 
-.-titleM {
+.-titleM-DEPRECATED {
 	@include titleM-DEPRECATED;
 }
 
-.-titleL {
+.-titleL-DEPRECATED {
```

Comment:
fixes some classess not being renamed in https://github.com/bethinkpl/design-system/commit/d83ce04b3f703e2a3df8560fa92803b2e272c18e